src/components/ui/blueprints/IndexPage.js

```diff
@@ -82,7 +88,11 @@ exports.waitForUploadDone = function( opts ){
 
 exports.deleteBlueprint = function(opts) {
     return exports.getBlueprint(opts).then(function(blueprint){
-        blueprint.element(by.css('.delete-button')).click();
-        return element(by.css('[ng-click="confirmDelete()"]')).click();
+        return  browser.executeScript('window.scrollTo(0, document.body.scrollHeight);').then(function(){
```

Comment:
Assuming one does have to scroll before openning the actions menus - this code is a duplicate of hard coded logic. it should be reused somehow. 

we should have a component "ActionsDropdown" which should expose a nice API to say "clickDefaultAction" and "clickMenuOption('delete')" or something. 

This component is going to be everywhere... BlueprintPage, DeploymentsIndex, DeploymentPage - it should be easy to test without copy-paste. copy-pastes are unacceptable.